src/pages/staking/deposit-contract.tsx

```diff
@@ -229,7 +229,7 @@ const DepositContractPage = ({
     // Create textToSpeechRequest
     let speech = new SpeechSynthesisUtterance()
     speech.lang = "en-US"
-    speech.text = DEPOSIT_CONTRACT_ADDRESS.split("").join(" ")
+    speech.text = DEPOSIT_CONTRACT_ADDRESS.split("").join(",")
```

Comment:
Nice. One note: this rapidly decreases the speed at which this is read, maybe we increase `speech.rate` to compensate?